user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---

```

Comment:
I believe it's part of YAML syntax, it's not displayed to the user and is used for categorizing the page.